.travis.yml

```diff
@@ -1,18 +1,25 @@
 language: python
 
+sudo: required
+
+services:
+  - docker
+
 cache:
   directories:
   - ~/.cache/pip
 
 python:
-- 2.7
 - 3.7
 
 install:
 - pip install -e .
+- docker-compose build
 
 script:
 - py.test tests
```

Comment:
The line `py.test tests` already runs inside the following command `docker-compose run test`.
This is defined in the `docker-compose.yaml` file, the services section defines a service called `test` and the command is defined here (https://github.com/GlobalFishingWatch/pipe-tools/blob/66-travis-drop-py27/docker-compose.yml#L6).
@pwoods25443 Could it be possible to remove this line?